src/weddingData.tsx

```diff
@@ -0,0 +1,63 @@
+export const weddings = [
+  {
+    weddingId: 1,
+    name: "Ghost",
+    image: "https://cdn0.weddingwire.com/articles/images/9/8/7/6/img_6789/r10_2x_s-wedding-day-timeline-kimberly-mufferi-photography.jpg",
+    date: "06/29/2021"
+  },
+  {
+    weddingId: 2,
+    name: "Johanessburg",
+    image: "https://i.guim.co.uk/img/media/21cf752c2469739f05db401e36e3c08fbfcb7c52/0_271_4000_2399/master/4000.jpg?width=1200&height=1200&quality=85&auto=format&fit=crop&s=7de9aa5231613602e57c86eb98a5ddb1",
+    date: "01/29/2021"
+  },
+  {
+    weddingId: 3,
+    name: "Kingsley",
+    image: "https://i.pinimg.com/236x/00/22/b3/0022b39fc82b2a7734b4285d4dd550bf.jpg",
+    date: "06/29/2020"
+  },
+  {
+    weddingId: 4,
+    name: "Bing",
+    image: "https://media-api.theknot.com/images/47215dcd-7470-43aa-99ef-05f1e9d14d3f",
+    date: "07/04/2020"
+  },
+  {
+    weddingId: 5,
+    name: "Bang",
+    image: "https://upload.wikimedia.org/wikipedia/commons/6/63/Hindu_wedding_rituals_b_%28cropped%29.jpg",
+    date: "02/14/2021"
+  },
+  {
+    weddingId: 6,
+    name: "Wong",
+    image: "https://hgtvhome.sndimg.com/content/dam/images/hgtv/fullset/2019/8/21/0/Christina_Anstead_Wedding_BrideGroom208.jpg.rend.hgtvcom.616.462.suffix/1566400786025.jpeg",
+    date: "01/04/2021"
+  }
+
+]
+
+export const individualWedding = {
+  weddingId: 2,
+  name: "Johanessburg",
+  image: "https://i.guim.co.uk/img/media/21cf752c2469739f05db401e36e3c08fbfcb7c52/0_271_4000_2399/master/4000.jpg?width=1200&height=1200&quality=85&auto=format&fit=crop&s=7de9aa5231613602e57c86eb98a5ddb1",
+  date: "01/29/2021",
+  email: "ghost@yahoo.com",
+  familyPhotoList: [
+    {name: "Bob", photos: [1, 2, 3, 4, 6], phone: 5555551234},
+    {name: "Mary", photos: [1, 2, 3, 4, 5, 6], phone: 5555554321},
+    {name: "Sue", photos: [1, 5, 6], phone: 5555550987},
+    {name: "John", photos: [3, 4, 5, 6], phone: 5555557890},
+    {name: "Jim", photos: [2, 5, 6], phone: 5555554567},
+    {name: "Gary", photos: [2, 5, 6], phone: 5555557654}
+  ],
+  photoList: [
+    {photoId: 1, guests: ["Bob", "Sue", "Mary"], description: "college buddies"},
+    {photoId: 2, guests: ["Gary", "Jim", "Mary", "Bob"], description: "The Out-laws"},
+    {photoId: 3, guests: ["John", "Bob", "Mary"], description: "The In-laws"},
+    {photoId: 4, guests: ["John", "Eric", "Brigette", "Bob"], description: "trouble"},
+    {photoId: 5, guests: ["Sue", "Gary", "John", "Jim", "Mary"], description: "old babysitters"},
+    {photoId: 6, guests: ["Bob", "Mary", "Sue", "John", "Jim", "Gary"], description: "this one's for mom"}
+  ]
+}
```

Comment:
This is great supplemental data to get our functions working correctly while the BE finishes